src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -1071,29 +1038,48 @@ namespace
 
         try
         {
-            if (extension == ".exr")
-            {
-                write_exr_image(
-                    bf_file_path,
-                    image,
-                    image_attributes);
+            bool high_dynamic_range_format = false;
+            if (extension == ".exr" || extension == ".tiff" || extension == ".tif" || extension == ".hdr")
```

Comment:
Even simpler:

    const bool high_dynamic_range_format =
        extension == ".exr" ||
        extension == ".tiff" ||
        extension == ".tif" ||
        extension == ".hdr";

Reply:
Thanks, I'll change it.